topics/MBC_cluster_conversion/mbc_divider_script.py

```diff
@@ -293,6 +293,20 @@ def main():
         run_shell_command(drain_s3_cmd)
         s3_drain_grace_period = int(args.s3_drain_gracetime)
         sleep(s3_drain_grace_period)
+        #validate drain
+        validate_drain_s3_cmd = '/bin/sh -c "weka debug jrpc container_get_drain_status hostId={}"'.format(current_host_id)
+        s3_drain_status = run_shell_command(validate_drain_s3_cmd).strip()
+        if s3_drain_status == 'false':
```

Comment:
omg you're comparing strings. This is horrible. At least do `s3_drain_status`.lower()